interface/src/app/greenelab/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * This module includes a few utility components that will be shared in other
+ * modules to avoid duplicate codes.
+ */
+
+angular.module('greenelab.utils', [])
+.constant('ApiBasePath', '/api/v0/')
```

Comment:
If you guys like `constants.js`, I can change `errGen` service from a **value** to **constant**. It means that later we can not use decorator to customize `errGen` service.